library/globals.inc.php

```diff
@@ -1615,7 +1615,12 @@
       '1',
       xl('New form validation')
      ),
-
```

Comment:
Place a global here called something like:
allow_multiple_databases

And then only run the seek for multiple databases if this is on.